jobs/build-node-image.Jenkinsfile

```diff
@@ -161,6 +164,11 @@ lock(resource: "build-node-image") {
                 }
             }
         }
+        stage("Brew Upload") {
```

Comment:
Please keep the `Release Manifests` the final one, as that indicates that everything else is good.

Reply:
Yeah, that makes sense to me.